pyiron_atomistics/atomistics/structure/pyscal.py

```diff
@@ -234,13 +223,63 @@ def analyse_voronoi_volume(atoms):
     Args:
         atoms : (pyiron_atomistics.structure.atoms.Atoms): The structure to analyze.
     """
-    s.publication_add(publication())
-    sys = pc.System()
-    sys.read_inputfile(atoms, format="ase")
+    sys = get_system(atoms)
     sys.find_neighbors(method="voronoi")
     atoms = sys.atoms
     return np.array([atom.volume for atom in atoms])
 
+def get_system(atoms):
```

Comment:
I think for consistency we would want to call that `pyiron_to_pyscal`, similar to `pyiron_to_ase` and `pyiron_to_pymatgen`.

Reply:
renamed it. I also moved it next to the to_ase, to_pymatgen and to_ovito methods